roles/static_inventory/tasks/sshtun.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Create ssh tunnel systemd service
+  template:
+    src: ssh-tunnel.service.j2
+    dest: /etc/systemd/system/ssh-tunnel.service
+    mode: 0755
```

Comment:
Oh, you've fixed it in the meantime, sorry! I get confused by the github ui sometimes. Will merge now.